.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.4.4-dev0
+current_version = 0.4.5-dev0
```

Comment:
you should bump to `dev1` here.

```
bump2version build
```

Reply:
done, why was this needed ?

Reply:
You keep bumping patch version without going through dev. We should at least release `0.4.4` before jumping to `0.4.5`